packages/row/src/react/__stories__/index.story.js

```diff
@@ -91,23 +96,19 @@ Object.keys(Row.sizes).forEach(size =>
       size={size}
       title={`${size} With the Longest Title in the World Because It's Known That If You Give Room For This Kind of a Title, It Will Definitely Come to Pass 2000`}
       metadata1={['Jim Cooper']}
-      image={
-        <Row.Image src="http://lorempixel.com/output/technics-q-c-680-320-6.jpg" />
-      }
+      image={<Row.Image src={getImgSrc()} />}
     />
   ))
 )
 
 const progressStory = storiesOf('progress', module)
-;[0, 66.6667, 100].forEach(progress =>
+Array.from([0, 66.6667, 100]).forEach(progress =>
```

Comment:
why copy the array here?

Reply:
i didn't like the leading `;`...thoughts?

Reply:
I love the leading semicolon.  Semicolons are not needed in most places, but they are here.  So the "love" comes from using it in the limited places it is needed.

Reply:
:) i'll revert. i mean when else are we going to get the chance to use a semi